test/node-unit/mocha.spec.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+var Mocha = require('../../lib/mocha');
+
+describe('.unloadFile()', function() {
+  it('should load and unload file to/from cache', function() {
+    var mocha = new Mocha({});
+    var filePath = __filename;
+    mocha.addFile(filePath);
+    mocha.loadFiles();
+
+    expect(
+      require.cache[require.resolve(filePath)] !== undefined,
+      'to be',
+      true
+    );
+    mocha.unloadFile(filePath);
+    expect(require.cache[require.resolve(filePath)], 'to be', undefined);
+  });
+});
+
+describe('.unloadFiles()', function() {
+  it('should unload all test files from cache', function() {
```

Comment:
Perhaps this test would be more meaningful if we added multiple files, then made the assertion they were all removed by looping through `mocha.files`.